src/js/extensions/paste.js

```diff
@@ -60,7 +60,10 @@
             [new RegExp(/<\/?o:[a-z]*>/gi), ''],
 
             // Microsoft Word adds some special elements around list items
-            [new RegExp(/<!\[if !supportLists\]>(((?!<!).)*)<!\[endif]\>/gi), '$1']
+            [new RegExp(/<!\[if !supportLists\]>(((?!<!).)*)<!\[endif]\>/gi), '$1'],
+
+            // remove font tag
+            [new RegExp(/<font[^><]*>|<.font[^><]*>/gi), '']
```

Comment:
So my main concern is that with this PR, it will filter all `<font>` tags from the paste for every user of medium-editor who upgrades.  I could see scenarios where this might not be desired so I'm a bit hesitant to merge this into the main repo as-is (I'm happy to hear feedback from others on this as well though).  

Most of the existing global replacements in the paste handler apply to elements that are very specific side-effects from things like Word or Google Docs.  `<font>` tags could come from all kinds of sources, like copying text from another webpage, so it would be better if there's a way to target just `<font>` tags that come from Word and not all `<font>` tags.  Medium-Editor itself injects `<font>` tags when you a trigger some of the native formatting commands like change font size, change font face, or change font color.  So, if you tried copy and pasting from the editor back into the same editor (or another instance) those font changes would be lost.

Were you aware that you can always pass in your own extra custom RegEx objects for cleaning the pasted html when you initialize MediumEditor?  You can do this by overriding some of the [paste options](https://github.com/yabwe/medium-editor#paste-options).

So for filtering out the `<font>` tags for your instance, you could do something like this:

```js
var editor = new MediumEditor('.editable', {
    paste: {
        cleanPastedHTML: true,
        cleanReplacements: [[new RegExp(/<font[^><]*>|<.font[^><]*>/gi), '']]
    }
});
```

That would ensure that your custom replacements would be executed on each paste as well as when calls to `editor.cleanPaste()` occur.